backend/src/server.js

```diff
@@ -23,10 +25,12 @@ app.use(express.static(path.join(process.cwd(), "../frontend/src")));
 // Passport middleware
 app.use(passport.initialize());
 
-app.use("/api/login", authRouter);
+app.use("/api/auth", authRouter);
 app.use("/api/posts", postRouter);
 app.use("/api/users", userRouter);
 app.use("/api/translate", translationRouter);
+app.get('/api/auth/login', login);
+app.get('/api/auth/logout', logout);
```

Comment:
I think you can delete these two lines since you configured the `authRouter` for the `/api/auth` endpoint and you are binding the `login` and `/logout` functions to the `/login` and `/logout` endpoints respectively in the `authRouter`, you don't need to define them again here